utilities/perf-plotter/plot-perf.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+# Plot memory and CPU usage captured by the container-stats container.
+#
+# The script can be run interactively by downloading the benchmark artifact
+# from GHA, extract it and run the following command:
+#   jq -s 'flatten' perf.json | ./plot-perf.py -
+
+import argparse
+import json
+import sys
+import datetime as dt
+
+import matplotlib.pyplot as plt
+import matplotlib.dates as md
+
+
+def bytes_to_float(bytes: str) -> float:
```

Comment:
[nit] this function can return None if the suffix doesn't match `B|Mi|Ki|MiB|KiB` - I think either we want to remove the `else` so we always try to do `float(bytes) / 1000000` if the other statements don't match, or explicitly throw an exception. It's not a _huge_ concern though, given how this script is used

There is also the ability to match on `Mi|Ki` which I think is also unwanted (but again, not huge concern given the inputs and the intended use)

Reply:
Yeah, I thought about raising a runtime exception, then realized I didn't really care enough to put the effort in 🤣 , I'll tweak it a bit.